README.md

```diff
@@ -6,19 +6,19 @@
 <!-- end title -->
```

Comment:
I left it as not generated so that it doesn't modify existing readme's since that would change the behaviour.
I would like it as default though. :)
Whats your thought on this @neilime ?